### ISSUE TITLE
What batch size do you use when training?

### ISSUE THREAD

**Issue by thesby** [VITA-Group/DeblurGANv2#93]
I find that the batch size in config.yaml is 1, but 1 is not good for batch normalization. So I want to know the batch size to use when training.

**Comment by FrankLinxzx** [comment]
> I find that the batch size in config.yaml is 1, but 1 is not good for batch normalization. So I want to know the batch size to use when training.

i use 8 but seems slower than 1 ,so use 1
